signer/core/cliui.go

```diff
@@ -241,8 +243,32 @@ func (ui *CommandlineUI) OnApprovedTx(tx ethapi.SignTransactionResult) {
 	}
 }
 
+func (ui *CommandlineUI) showAccounts() string {
+	accounts, err := ui.api.ListAccounts(context.Background())
+	if err != nil {
+		fmt.Print("error listing accounts", err)
+	}
+	var resp string
+	if len(accounts) == 0 {
+		resp = "No accounts known to Clef"
+	} else {
+		// account info to string for nicer printing
+		var addresses string = "\nAccounts known to Clef:\n"
+		for i, account := range accounts {
+			// concat string to avoid repeating "INFO" on terminal
+			addresses += fmt.Sprintf("Account %v: %s at %s\n", i, account.Address, account.URL)
+		}
+		resp = addresses
+	}
+	return resp
+}
+
 func (ui *CommandlineUI) OnSignerStartup(info StartupInfo) {
-	fmt.Printf("------- Signer info -------\n")
+	go func() {
+		addresses := ui.showAccounts()
+		fmt.Printf("%s", addresses)
+	}()
```

Comment:
If you change the semantics of `showAccounts` so it actually displays the info by itself (and not return anything), then this can be minimized into

```suggestion
	go ui.showAccounts()
```

Reply:
oh, yes that is much nicer. Done in https://github.com/ethereum/go-ethereum/pull/26082/commits/4131dddf29c138f224bf91b9d7d9a48774f51b53